easybuild/easyconfigs/a/Autoconf/Autoconf-2.69-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'ConfigureMake'
+
+name = 'Autoconf'
+version = '2.69'
+
+homepage = 'https://www.gnu.org/software/autoconf/'
+
+description = """
+ Autoconf is an extensible package of M4 macros that produce shell scripts
+ to automatically configure software source code packages. These scripts can
+ adapt the packages to many kinds of UNIX-like systems without manual user
+ intervention. Autoconf creates a configuration script for a package from a
+ template file that lists the operating system features that the package can
+ use, in the form of M4 macro calls.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['954bd69b391edc12d6a4a51a2dd1476543da5c6bbf05a95b59dc0dd6fd4c2969']
+
+builddependencies = [
+    ('binutils', '2.34'),
```

Comment:
this also depends on a recent perl see https://github.com/easybuilders/easybuild-easyconfigs/issues/8704

Reply:
Good point, we should indeed move to including `Perl` as a build dep here...

Reply:
fixed in 7a16d59d7c